zipline/pipeline/data/dataset.py

```diff
@@ -119,5 +140,5 @@ def __repr__(self):
         return '<DataSet: %r>' % self.__name__
 
 
-class DataSet(with_metaclass(DataSetMeta)):
+class DataSet(with_metaclass(DataSetMeta, object)):
```

Comment:
yeah, I believe calling `type('name', (), {})` creates an old style class. At least it excludes `object` from the `__mro__`.